### PR TITLE
GSCHED-767: Fix timing window parsing and FT note parsing

### PR DESCRIPTION
**Pull request by bryanmiller**
Fix the setting of start times for OCS timing windows 
Improve the parsing of FT notes for setting the program start/stop dates.

Requires
https://github.com/gemini-hlsw/lucupy/pull/114

### CODE REVIEW

**Review by stroncod — COMMENTED** [review]
Some changes apply. Also the ocsenv pickle is deleted, it is created again? I don't see any changes to env so I don't get why is deleted

File: scheduler/core/programprovider/ocs/ocsprogramprovider.py [hunk 1]
```diff
@@ -5,11 +5,13 @@
 import json
 import zipfile
 from datetime import datetime, timedelta
+from astropy.time import Time
 from os import PathLike
 from pathlib import Path
 from typing import FrozenSet, Iterable, List, Mapping, Optional, Tuple, Dict
 
 import numpy as np
+from celery.utils.time import utcoffset
```
[the remark this answers is not in the document]
Comment (bryanmiller) [reply scheduler/core/programprovider/ocs/ocsprogramprovider.py:14]: I don't know enough about this.

File: scheduler/scripts/run_gsched767.py [hunk 2]
```diff
@@ -0,0 +1,120 @@
+# Copyright (c) 2016-2024 Association of Universities for Research in Astronomy, Inc. (AURA)
```
[the remark this answers is not in the document]
Comment (bryanmiller) [reply scheduler/scripts/run_gsched767.py:1]: This is no longer needed since I can run the scheduler again in Jupyter lab. I'll remove it.

**Comment by bryanmiller** [comment]
The ocsenv pickle does get recreated if it does not exist. I didn't delete it so it must not have been there the last time that I fetched from upstream.

**Pull request merged**